suites/test_search_beta.py

```diff
@@ -0,0 +1,64 @@
+import sys
+from selenium.webdriver.common.by import By
+import lemoncheesecake.api as lcc
+from lemoncheesecake.matching import *
+from helpers import utilities, locators, constants
+from helpers.base_screenshot import Screenshot
+from pages import search_beta_page
+
+
+@lcc.suite("Suite: Tests for search beta", rank="10")
+class test_search_beta(Screenshot):
+    driver = lcc.inject_fixture("driver_obj")
+
+    @lcc.test("Verify that warning message should be displayed when none of the repo is selected, verify toggle buttons")
+    def no_repo_selected(self):
+        utilities.click_element(self.driver, By.LINK_TEXT, locators.MENU_SEARCH_BETA_TEXT)
+        utilities.click_element(self.driver, By.ID, locators.TOGGLE_ID)
+        utilities.wait(1)
+        utilities.click_element(self.driver, By.ID, locators.TOGGLE_ID)
+        utilities.wait(1)
+        check_that("Filter by repo section is displayed", utilities.find_element(self.driver, By.CLASS_NAME,
+                                locators.FILTER_BY_REPO_SECTION_CLASS_NAME).is_displayed(), is_true())
+        check_that("no results found warning message", utilities.get_text(self.driver, By.CSS_SELECTOR,
+                                                                          locators.NO_RESULTS_FOUND_CSS),
+                   contains_string(constants.no_results_found))
+        utilities.click_element(self.driver, By.XPATH, locators.FILTER_BY_REPO_TOGGLE_XPATH)
+        utilities.wait(1)
+        utilities.click_element(self.driver, By.XPATH, locators.FILTER_BY_REPO_TOGGLE_XPATH)
+        utilities.wait(1)
+        check_that("verify repo list is displayed after expanding Filter by Repo",
+                   utilities.find_element(self.driver, By.XPATH,
+                                          locators.REPO_LIST_XPATH).is_displayed(), is_true())
+
+    @lcc.test("verify user is able to search repo")
+    def search_for_repo(self):
+        utilities.enter_text(self.driver, By.XPATH, locators.FILTER_BY_REPO_SEARCH_BAR_XPATH, "uploader")
+        utilities.wait(1)
+        check_that("entered repo name is displayed on search results",
+                   utilities.get_text(self.driver, By.XPATH, locators.SEARCH_REPO_XPATH),
+                   contains_string(constants.repo_name))
+        utilities.wait(1)
+        utilities.find_element(self.driver, By.CLASS_NAME, locators.CANCEL_BUTTON_ON_REPO_SEARCH_BAR_CLASS_NAME).click()
+
+    @lcc.test("verify user is able to select repo")
```

Comment:
@pbadak along with other suggested changes, here is the one we discussed in detail:
- after you select a repo, capture the modules listed in a variable.
- modules_displayed = find all elements by: data-label="Title"  This is the unique one I could find right away.
- use a for loop to iterate over the list:
```
for i in moduled_displayed:
  check that i.text contains_string (Module prefix used as per the test data)
```
This will ensure that the modules listed are from the test repo itself when you select the repo.
@noopurmath3 please suggest if otherwise. Thanks.
@pbadak feel free to reach out if you have any questions.

Reply:
Incorporated all review comments :)

Reply:
awesome! thanks!